table/tables/tables.go

```diff
@@ -1019,6 +1015,14 @@ func GetColDefaultValue(ctx sessionctx.Context, col *table.Column, defaultVals [
 
 // AllocHandle allocate a new handle.
 func AllocHandle(ctx sessionctx.Context, t table.Table) (int64, error) {
+	if stmtCtx := ctx.GetSessionVars().StmtCtx; stmtCtx != nil {
```

Comment:
Take care of some test, `ctx` maybe nil here, better check it.

Reply:
Done